aiostream/stream/misc.py

```diff
@@ -32,15 +39,15 @@ async def ainnerfunc(arg: T, *_: object) -> T:
             await awaitable
             return arg
 
-        return map.raw(source, ainnerfunc)
+        return map.raw(source, ainnerfunc, ordered=True, task_limit=task_limit)
```

Comment:
Did you consider allowing `ordered` to be specified too?

Reply:
I don't know, isn't it weird to have `action(ordered=False)` changing the order of the items depending on how long the action took? 

Maybe not, I can definitely expose it if you find it useful though.

Reply:
I think you could argue the same for `map`, and the ordered=False is supported over there.

I have a use-case where ordering isn't required. Disabling ordering improves performance by getting rid of head-of-line blocking.

Reply:
Alright, updated :+1: